2023.11 update for Withings (#29510)
home-assistant/home-assistant.github.io: source/_integrations/withings.markdown

> The integration will automatically detect if you can use webhooks. This will enable the integration only to update when there is new data.

Reviewer: ```suggestion
The integration automatically detects if you can use webhooks. This enables the integration only to update when there is new data.
```

The binary sensor for sleep will only work if the integration can establish webhooks with Withings.

## Available data

> The integration provides several entities, some of which dynamically enabled based on the data present.

Reviewer: ```suggestion
The integration provides several entities, some of which are dynamically enabled if data is available.
```

> For example, measurement senors like weight will only work when there has been data registered for it in the last 14 days. So if you start using a new device, for example something to measure your temperature or you manually update a value in the app, the sensor will automatically appear.

Reviewer: ```suggestion
For example, measurement sensors like weight only work when data has been registered in the last 14 days. So if you start using a new device, for example, to measure your temperature or you manually update a value in the app, the sensor automatically appears.
```

> Sleep sensors will only be created if the integration can find sleep data for you in the last day.

Reviewer: ```suggestion
Sleep sensors are only created if the integration can find sleep data for you within the last day.
```

> For the workout calendar and the workout and activity sensors, it's the same, but 14 days.

Reviewer: ```suggestion
Workout calendar and the workout and activity sensors show if the latest available data point is no older than 14 days.
```

Reviewer: just an idea. Feel free to ignore.

Author: but this is also covering the workout calendar, hmmm
